.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   build:
     name: Build and analyze
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Yes, the android-emulator used is apparently worse on ubuntu, since it doesn't use [hardware acceleration](https://github.com/ReactiveCircus/android-emulator-runner#can-i-use-this-action-on-github-hosted-linux-vms). And switching didn't seam to change anything else, so one upside and no downsides.

Reply:
Nice thing 😎